federation-js/src/composition/validate/postComposition/__tests__/keyFieldsSelectInvalidType.test.ts

```diff
@@ -113,12 +111,12 @@ describe('keyFieldsSelectInvalidType', () => {
     };
 
     const serviceB = {
-      typeDefs: parse(`
+      typeDefs: gql`
```

Comment:
I haven't decided yet if it's unexpected that there's no change in locations for this commit. Will revisit, just scanning for now.